docs/source/asr/speaker_diarization/intro.rst

```diff
@@ -19,14 +21,18 @@ The following figure shows the overall data flow of the NeMo speaker diarization
 
 NeMo speaker diarization system consists of the following modules:   
 
-**Voice Activity Detection (VAD)**: A trainable model which detects the presence or absence of speech in the given audio recording.  
-**Speaker Embedding Extractor**: A trainable model that extracts speaker embedding vectors containing voice characteristics from raw audio signal.   
-**Clustering Module**: A non-trainable module that groups speaker embedding vectors into a number of clusters.   
-**Neural Diarizer**: A trainable model that estimates speaker labels from the given features.   
+- **Voice Activity Detection (VAD)**: A trainable model which detects the presence or absence of speech in the given audio recording.  
```

Comment:
since it's diarization, how about change it from "detects the presence or absence of speech in the given audio recording.  "  to thing like "detects the presence or timestamp of speech in the give recording"?

Reply:
Reflected comments